packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardFilterBuilder.tsx

```diff
@@ -230,8 +230,10 @@ export const CloudPulseDashboardFilterBuilder = React.memo(
     );
 
     const toggleShowFilter = () => {
-      setShowFilter((showFilterPrev) => !showFilterPrev);
```

Comment:
handleToggleAppliedFilter(showFilter);
setShowFilter((showFilterPrev) => !showFilterPrev);

I think this can just be interchanged...